lib/swaps/Swaps.ts

```diff
@@ -739,7 +739,8 @@ class Swaps extends EventEmitter {
         source = 'Taker';
         destination = 'Maker';
         const lockExpirationDelta = expiration - chain_height;
-        if (deal.makerCltvDelta! > lockExpirationDelta) {
+        const LOCK_EXPIRATION_SLIPPAGE = 3;
+        if (deal.makerCltvDelta! - LOCK_EXPIRATION_SLIPPAGE > lockExpirationDelta) {
```

Comment:
consider to add the LOCK_EXPIRATION_SLIPPAGE to the maker's final_cltv_delta  and here just check without the LOCK_EXPIRATION_SLIPPAGE.

Reply:
I don't understand. In the case of raiden there's no concept of `final_cltv_delta`. The way I understand it is that we only want to add the slippage for raiden payments from taker to maker.